src/inspector_agent.cc

```diff
@@ -170,24 +173,28 @@ class AgentImpl {
   void WaitForDisconnect();
 
  private:
+  using MessageQueue =
+    std::vector<std::pair<int, const blink::protocol::String16>>;
```

Comment:
Just a suggestion but you could add a `using blink::protocol::String16;` at the top.